utoipa-swagger-ui/build.rs

```diff
@@ -24,6 +31,11 @@ const SWAGGER_UI_DOWNLOAD_URL_DEFAULT: &str =
 const SWAGGER_UI_DOWNLOAD_URL: &str = "SWAGGER_UI_DOWNLOAD_URL";
 const SWAGGER_UI_OVERWRITE_FOLDER: &str = "SWAGGER_UI_OVERWRITE_FOLDER";
 
+// wget <url> && sha256sum <file> | tr '[a-z]' '[A-Z]'
+#[cfg(feature = "cache")]
+const SWAGGER_UI_FILE_HASH: &str =
```

Comment:
But what happens if the user changes the url? I guess the validation will then just fail?

Reply:
Ah, it is only used as a suffix, well this is not very good to be hard coded because users can change the `SWAGGER_UI_DOWNLOAD_URL` which effectively is changing the version that changes the hash as well.

Reply:
> Ah, it is only used as a suffix, well this is not very good to be hard coded because users can change the `SWAGGER_UI_DOWNLOAD_URL` which effectively is changing the version that changes the hash as well.

I think we should avoid use cache if user supplied the URL